<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDashSpaceWidgetBase</class>
 <widget class="QgsPanelWidget" name="QgsDashSpaceWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dash Space Pattern</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="mAddButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mRemoveButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>245</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="mDashSpaceTreeWidget">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Dash</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Space</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pattern length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPatternLengthLabel">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
